index.html

```diff
@@ -58,209 +58,5 @@ <h3 class="hero-highlight-2">I’am a Software Developer</h3>
         />
```

Comment:
- [ ] Kindly add the background image using the `URL()` role in CSS, because here the image is not an element, it's more semantic.